x-pack/legacy/plugins/lens/public/indexpattern_plugin/datapanel.tsx

```diff
@@ -258,9 +260,8 @@ export const InnerIndexPatternDataPanel = function InnerIndexPatternDataPanel({
 
     return true;
   });
-
-  const paginatedFields = displayedFields.sort(sortFields).slice(0, pageSize);
-
+  const paginatedFields = displayedFields.sort(sortFields).slice(0, pageSize - 1);
+  paginatedFields.unshift(documentField);
```

Comment:
I think it's fine to treat the document as a special field tyoe here to re-use infrastructure where possible.

But we should take care of the flyout:
<img width="696" alt="Screenshot 2019-10-08 at 09 34 48" src="https://user-images.githubusercontent.com/1508364/66375964-e78efd80-e9ae-11e9-9fe8-d361e13e0660.png">

Currently it's just saying that no data is available, IMHO this should be a regular tooltip (`EuiTooltip`) instead showing some explanation what this field means.